Components/CreateTrip/CreateTrip.js

```diff
@@ -1,39 +1,145 @@
 
-export const CreateTrip = () => {
+export const CreateTrip = ({ route }) => {
   return (
     <View style={styles.container}>
-      <Text>Trip name:</Text>
-      <TextInput style={styles.input}
-        placeholder='e.g. Spring break'
-      />
-      <Text>Starting Location:</Text>
-      <TextInput style={styles.input}
-        placeholder='e.g. trip origin'
-      />
-      <Text>Destination:</Text>
-      <TextInput style={styles.input}
-        placeholder='e.g. trip destination'
-      />
-      <TouchableOpacity>
+      <View style={styles.header}>
+        <Image
+          style={styles.image}
+          source={{uri: 'https://img.icons8.com/officel/40/000000/globe.png'}}
+        />
+        <Text style={{fontSize: 20, marginLeft: 33}}>
+          New Trip
+        </Text>
+        <Text style={{color: '#96CDEC', marginRight: 1}}>
+          placehold
+        </Text>
+      </View>
+      <View>
+        <Text>Trip name:</Text>
+        <TextInput style={styles.input}
+          placeholder='e.g. Spring break'
+        />
+      </View>
+      <View>
+        <Text>Starting Location:</Text>
+        <TextInput style={styles.input}
+          placeholder='e.g. trip origin'
+        />
+      </View>
+      <View>
+        <Text>Destination:</Text>
+        <TextInput style={styles.input}
+          placeholder='e.g. trip destination'
+        />
+      </View>
+      <View style={styles.legContainer}>
+        <View style={styles.calendars}>
+          <View style={{textAlign: 'center'}}>
+            <Text>Start:</Text>
+            <Image
+              style={{height: 60, width: 60}}
+              source={{uri: 'https://img.icons8.com/plasticine/200/000000/calendar.png'}}
+            />
+          </View>
+          <View style={{textAlign: 'center'}}>
+            <Text>End:</Text>
+            <Image
+              style={{height: 60, width: 60}}
+              source={{uri: 'https://img.icons8.com/plasticine/200/000000/calendar.png'}}
+            />
+          </View>
+        </View>
+        <TouchableOpacity style={styles.buttonLeg}>
+          <Text>Add Trip Leg</Text>
+        </TouchableOpacity>
+      </View>
+      <TouchableOpacity style={styles.buttonDestination}>
         <Text>Add destination</Text>
       </TouchableOpacity>
     </View>
   )
 }
 
-export default CreateTrip;
+
 
 const styles = StyleSheet.create({
   container: {
-
+    alignItems: 'center',
+    backgroundColor: '#96cdec',
+    borderRadius: 4,
+    borderWidth: 1.5,
+    borderColor: 'black',
+    flex: 1,
+    height: '100%',
+    justifyContent: 'space-around',
+    padding: 20,
+    width: '100%'
+  },
+  header: {
+    backgroundColor: '#96cdec',
+    borderBottomWidth: 1.5,
+    borderBottomColor: 'black',
+    fontSize: 30,
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    paddingBottom: 10,
+    width: '100%'
+  },
+  image: {
+    height: 36,
+    width: 36,
```

Comment:
I'm not 100% on this but I think the browser is trying to emulate a mobile device we might want to bump the width up a little and act as though this is what it will look like on a phone.  I am still trying to get this work on the android previewer ( and will look into the apple one as well ).

Reply:
Appreciate the feedback!